test/test.js

```diff
@@ -172,6 +173,23 @@ test.serial('pack path that does not exist (async)', async t => {
     t.true(hasWrongPathMentioning);
 });
 
+test.serial('custom binary path', async t => {
+    const settings = _7z.getConfig();
+    _7z.config({
+        binaryPath: CUSTOM_BINARY_PATH
+    });
+    
+    const output = (await t2p(cb => {
+        _7z.unpack(ARCH_PATH, UNPACK_PATH, cb);
+    }))[0];
+
+    _7z.config(settings);
+
+    const unpackSrcPath = join(UNPACK_PATH, SRC_DIR_NAME);
+    t.deepEqual(...await getFilesList(unpackSrcPath));
+    t.is(typeof output, 'string');
+});
+
```

Comment:
Could we also have a test like "wrong custom binary path"?
To make sure that error is thrown correctly in case of a wrong custom path?